karma.conf.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(config) {
 
 		// list of files / patterns to load in the browser
 		files: [
-			'http://cdn.polyfill.io/v2/polyfill.js?features=fetch&flags=gated',
+			'https://cdn.polyfill.io/v2/polyfill.js?flags=gated&ua=safari/4&features=default',
```

Comment:
I don't think so, I think the initial setup was copied from one that did, but by default we shouldn't include it